.travis.yml

```diff
@@ -8,6 +8,10 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.6-dev"
+  - "3.7-dev"
+  - "nightly"
```

Comment:
The article actually specifically suggests *against* including nightly in your testing because of its instability.